packages/audio-block/src/components/AudioPlayer.tsx

```diff
@@ -51,6 +51,7 @@ export const AudioPlayer = ({
                 <LoadingCircle />
             </div>
         ) : (
+            // eslint-disable-next-line jsx-a11y/media-has-caption
```

Comment:
Can we turn this rule off in the eslintrc?

Reply:
In my opinion this rule has a point and we should check with martin how we wanna implement the option to add caption tracks in the audio block and later in the video block